### PR TITLE
Various improvements/fixes for v1

### PR DESCRIPTION
**Pull request by jhesketh**
This fixes issues I was seeing with Xiaomi Mi Robot Vacuum Cleaner (v1) running Valetudo (0.9.4).

### CODE REVIEW

File: src/vacuum-card.js [hunk 1]
```diff
@@ -351,9 +355,14 @@ class VacuumCard extends LitElement {
...
-      this.entity
-    );
+    const {
+      status,
+      battery_level,
+      battery_icon,
+      valetudo_state,
+    } = this.getAttributes(this.entity);
+
+    const val_status = status || valetudo_state['name'];
```
Comment (denysdovhan) [inline src/vacuum-card.js:365]: Can we hide this platform-specific properties, like `valetudo_state` under `getAttribute` method?

Comment (jhesketh) [reply src/vacuum-card.js]: Good point. Done.

**Comment by denysdovhan** [comment]
Thanks for contributing!
Published as `0.3.4`!

**Comment by denysdovhan** [comment]
Please, note breaking changes introduced by #29. It will be released as 1.0.0.

**Pull request merged**